data-prepper-plugins/kafka-source/src/test/java/org/opensearch/dataprepper/plugins/kafka/source/KafkaSourceConfigTest.java

```diff
@@ -0,0 +1,80 @@
+package org.opensearch.dataprepper.plugins.kafka.source;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.opensearch.dataprepper.plugins.kafka.source.configuration.ConsumerConfigs;
+import org.opensearch.dataprepper.plugins.kafka.source.configuration.TopicsConfig;
+import org.yaml.snakeyaml.Yaml;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.opensearch.dataprepper.plugins.kafka.source.configuration.SchemaConfig;
+
+class KafkaSourceConfigTest {
+
+	@Mock
```

Comment:
This Mock annotation seems redundant since you are instantiating real instance in the setup